src/toxcore/dht/kbucket.rs

```diff
@@ -477,15 +501,20 @@ impl Kbucket {
     }
 
     /// Create iterator over [`DhtNode`](./struct.DhtNode.html)s in `Kbucket`.
+    /// Nodes that this iterator produces are sorted by distance to a base
+    /// `PublicKey` (in ascending order).
     pub fn iter(&self) -> impl Iterator<Item = &DhtNode> {
         self.buckets.iter()
+            .rev()
```

Comment:
Why rev?

Reply:
Because kbucket stores farther nodes at the beginning, but bucket - at the end.